src/main/scala/io/sdkman/changelogs/JqaMigrations.scala

```diff
@@ -0,0 +1,36 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+import org.bson.Document
+
+@ChangeLog(order = "089")
+class JqaMigrations {
+  final val CandidateName = "jqassistant"
+
+  @ChangeSet(order = "001", id = "001_add_jqa_2_6_0", author = "ascheman")
+  def migration001(implicit db: MongoDatabase): Document = {
+    Candidate(
+      candidate = CandidateName,
+      name = "jQAssistant",
+      description =
+        "jQAssistant is an open-source tool for software analytics, enabling you to gain insights into your systems, verify design implementation, and create living documentation. By scanning code and configurations into a Neo4j graph database, it helps maintain software quality and bridge the gap between documentation and implementation.",
+      websiteUrl = "https://jqassistant.org"
+    ).insert()
+    val v250 = Version(
```

Comment:
I'd suggest not inserting the versions here but doing this using the API. I'd delete everything from line 20 down.

Reply:
I followed your recommendation.

However, I wonder why you requested this change as some weeks ago you [recommended more or less the opposite](https://github.com/sdkman/sdkman-db-migrations/pull/719#issuecomment-2657421152), i.e., why I didn't directly add available versions when creating HSC (cf. #719).

Reply:
That's odd, not sure why I suggested that. I generally see the first versions as a great exercise to treat API access.